### ISSUE TITLE
Other error occurred: record_invalid Invalid parameters

### ISSUE THREAD

**Issue by qbixx** [cyberjunky/3commas-cyber-bots#28]
**Describe the bug**
When running the compound.py script, the error below shows in the logs when it's trying to update the BO/SO.

**To Reproduce**
1. Which program you run
compound.py

2. Relevant settings used (don't paste your bot-id's, or any api-key values!)
```
[settings]
timezone = Europe/Amsterdam
timeinterval = 3600
debug = False
logrotate = 7
botids = [1234, 12345]
profittocompound = 0.9
3c-apikey = [...]
3c-apisecret = [...[
notifications = False
notify-urls = ['notify-url1', 'notify-url2']
```

2. Error you get
```
2021-11-27 19:27:13 - compound.py - ERROR - Error occurred updating bot with new BO/SO values: Other error occurred: record_invalid Invalid parameters {'pairs': ['No market data for this pair: USDT_AAVEDOWN', 'No market data for this pair: USDT_AAVEUP', 'No market data for this pair: USDT_SXPDOWN', 'No market data for this pair: USDT_SXPUP']}.
```

**Environment:**
 - Python version: Python 3.8.10
 - Operating System: Ubuntu 20.04
 - Version of software [e.g. 0.6.1]: 0.7.9

**Comment by cyberjunky** [comment]
These pairs are all UP/DOWN pairs, not sure how that works with market data.
Which exchange do you use?

**Comment by qbixx** [comment]
@cyberjunky I'm using Binance. I don't know if it's important, but i've put all UP/DOWN pairs to my blacklist in 3commas.

**Comment by cyberjunky** [comment]
@qbixx You are having this issue with compound right? 
I don't check pairs in compound.py, since I don't change them, I only place back the pairs that the given bots already had during updating, seems a 3Commas bug to me. For now, I guess you should remove them manually from the bot(s)

**Comment by qbixx** [comment]
@cyberjunky Thanks for pointing me in the right direction. I've found the root cause. The pairs that gave the error where added to my blacklist. However these coins aren't available anymore on Binance. I just removed them from my blacklist and the error disappeared. 3commas apparently checks both the coins that are inside the bot, but also the ones that are on the blacklist. If a coin isn't available anymore they are returning the error like above.
